qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -48,6 +48,18 @@
                             expect_items = 48
                             cpu_model_flags = ",-la57"
                             no_flags = "la57"
+                - pt_vmx:
+                    type = x86_cpu_flag_intel_pt
+                    flags = "intel_pt"
+                    required_qemu = [4.2.0, )
+                    start_vm =no
+                    auto_cpu_model = yes
```

Comment:
It's yes by default, no need to add this line.

Reply:
Thanks and update.